docs/source/tutorials/partition_table.rst

```diff
@@ -0,0 +1,188 @@
+###############
+Partition Table
+###############
+
+Introduction
+------------
+
+Partition table is used to define the flash memory organization and the different kind of data will be stored on each partition.
+
+You can use one of the predefined partition table schemes available or create your own.
+
+The partition table is created using a CSV file format with the following format:
+
+.. code-block::
+
+    # ESP-IDF Partition Table
+    # Name, Type, SubType, Offset, Size, Flags
+
+Where:
+
+1. **Name**
+
+    Is the partition name and must be a unique name. This name is not relevant for the system and the size must be at maximum of 16-chars.
+
+2. **Type**
+
+    This is the type of the partition. This value can be ``data`` or ``app``.
+
+    * ``app`` type is used to define the partition that will store the application.
+
+    * ``data`` type can be used to define the partition that stores general data, not the application.
+
+3. **SubType**
+
+    The SubType defines the usage of the ``app`` and ``data`` partitions.
+
+   **data**
```

Comment:
Are some of subtypes mandatory? I see some are optional (maybe we can highlight this more?)


_This is maybe not valid questions as more experienced Devs are going to add their custom partition table scheme._

Reply:
It will depend if you need OTA or not.